.github/workflows/cleanup-pr-registry-images.yaml

```diff
@@ -0,0 +1,69 @@
+name: Cleanup PR Registry Images
+on:
+  pull_request:
+    types:
+      - closed
+  inputs:
+    webImage:
```

Comment:
I am not sure if we have a use case to have solr here, besides maybe arce for the solr image. I need to look at that repo and see if it really needs to have a solr image built every time or if I could set it up just like our hyku solr images where they are run when you manually run it.